### PR TITLE
fix #100101: pre-populate Album Manager with default "Untitled"

### PR DESCRIPTION
**Pull request by vinayakvivek**
- Create an empty album with name as "Untitled" when Album Manager window opens up.
- On reopening album manager focus on album name textbox

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
As the Album feature itself is gone from the master branch since  902a5f7 , I guess this PR is no longer needed nor feasible, so most probably should get closed.
Maybe it could go into 2.1 though?

**Comment by IsaacWeiss** [comment]
It would be a shame not to put it in 2.1 with the work done.

**Comment by Jojo-Schmitz** [comment]
Indeed and very true, esp. as it still applies cleanly even to master and is reasonably small

File: mscore/albummanager.cpp [hunk 1]
```diff
@@ -60,6 +60,14 @@ AlbumManager::AlbumManager(QWidget* parent)
...
 
       MuseScore::restoreGeometry(this);
+          
+      if (album == 0) {
+          createNewClicked();
+          }
+          
+      if (albumName->text().isEmpty()) {
+          albumName->setEnabled(true);
+          }
```
Comment (lasconic) [inline mscore/albummanager.cpp]: We use 6 spaces tab. See https://musescore.org/en/developers-handbook/musescore-coding-rules#Indentation-%26-Whitespaces

File: mscore/albummanager.cpp [hunk 2]
```diff
@@ -60,6 +60,14 @@ AlbumManager::AlbumManager(QWidget* parent)
       createScore->setEnabled(false);
 
       MuseScore::restoreGeometry(this);
+          
+      if (album == 0) {
```
Comment (lasconic) [inline mscore/albummanager.cpp]: I don't this this test is needed. Album is 0, we just set it to 0 no?

File: mscore/albummanager.cpp [hunk 3]
```diff
@@ -60,6 +60,14 @@ AlbumManager::AlbumManager(QWidget* parent)
...
+          createNewClicked();
+          }
+          
+      if (albumName->text().isEmpty()) {
```
Comment (lasconic) [inline mscore/albummanager.cpp]: I don't get the purpose of this block. Could you comment your code?

Comment (Jojo-Schmitz) [inline mscore/albummanager.cpp]: I think at that spot, in the Default constructor, it is always empty?

**Comment by Jojo-Schmitz** [comment]
For some strange reason Travis didn't try to build this. Had it tried, it would have failed (on master), and it does now.

**Pull request merged**